witchcraft-server-ete/Cargo.toml

```diff
@@ -28,6 +28,9 @@ http-body-util = "0.1"
 hyper-util = { version = "0.1", features = ["tokio"] }
 hyper = "1"
 libc = "0.2"
-openssl = "0.10"
```

Comment:
Easier to just switch the ETE tests over to using rustls for their client rather than getting openssl built/linked on windows.